src/wrappers/themis/react-native-themis/package.json

```
@@ -0,0 +1,157 @@
+{
+  "name": "react-native-themis",
+  "version": "0.14.0",
```

```suggestion
  "version": "0.14.1",
```

i think we already have `0.14.0`